### PR TITLE
os/board/rtl8730e To start softap on wlan1

### PR DESCRIPTION
**Pull request by Wuth5**

### CODE REVIEW

**Review by a-jhanwar — CHANGES_REQUESTED** [review]
Please share your opinion on review comment.

**Comment by Wuth5** [comment]
hi abhishek,
yes you are correct, typo here, I will remove the debug log also
besides please also help to check if any flag was set incorrectly in [rtk_netmgr.c] also

File: os/board/rtl8730e/src/component/os/tizenrt/rtk_netmgr.c [hunk 1]
```diff
@@ -361,19 +361,33 @@ trwifi_result_e wifi_netmgr_utils_init(struct netdev *dev)
...
-			return wuret;
-		}
-		g_mode = RTK_WIFI_STATION_IF;
-		/*extern const char lib_wlan_rev[];
-		RTW_API_INFO("\n\rwlan_version %s\n", lib_wlan_rev);*/
-		wuret = TRWIFI_SUCCESS;
-		softap_flag = 0;
 		rtw_mutex_init(&scanlistbusy);
-	} else {
+	} else if(rtw_memcmp(dev->ifname,"wlan1",4)){
```
[the remark this answers is not in the document]
Comment (Wuth5) [reply os/board/rtl8730e/src/component/os/tizenrt/rtk_netmgr.c]: yes you are correct, typo here, I will fix it and besides will remove the debug log
please also help to check if any flag was set incorrectly in [rtk_netmgr.c]

**Comment by Wuth5** [comment]
hi abhishke, I have revised the code as per requested, kindly help to review again, thanks

**Review by a-jhanwar — APPROVED** [review]
Changes looks fine to me.

**Pull request merged**